charts/schema-registry/templates/deployment.yaml

```diff
@@ -41,9 +41,11 @@ spec:
               protocol: TCP
           livenessProbe:
             initialDelaySeconds: 2
-            httpGet:
-              path: /ping
-              port: rest
+            exec:
+              command:
+                - sh
+                - -c
+                - netstat -ltn 2> /dev/null | grep {{ .Values.service.port }} || ss -ltn 2> /dev/null | grep {{ .Values.service.port }}
```

Comment:
Here we are only checking that the port is up or not. This seems to be not enough we should check liveness using HTTP(in case tls is disabled) and using HTTPS(in case tls is enabled) as there can be cases where port might be up but tls is not enabled.

Reply:
That is something that needs to be tested by the Readiness Probe. The purpose of the Liveness Probe however is just to understand when the container is up and responsive or not.